### ISSUE TITLE
Missing SettingsPatch

### ISSUE THREAD

**Issue by owen-the-kid** [crimera/piko#298]
i am trying to use the patches on the compose manager and it says its missing a dependency called SettingsPatch

**Comment by Mylloon** [comment]
Yeah the Settings patch isn't applied using the Revanced Manager Compose

**Comment by owen-the-kid** [comment]
> Yeah the Settings patch isn't applied using the Revanced Manager Compose, try using the other for now
i am doing that for now

**Comment by owen-the-kid** [comment]
i am doing that

**Comment by just-ero** [comment]
Running into the same issue, would love to know how to solve this.

**Comment by krvstek** [comment]
This should be closed as it is outdated.
